lib/features/mailbox_dashboard/data/datasource/session_storage_composer_datasource.dart

```diff
@@ -1,10 +1,12 @@
+import 'package:jmap_dart_client/jmap/account_id.dart';
+import 'package:jmap_dart_client/jmap/core/user_name.dart';
 import 'package:jmap_dart_client/jmap/mail/email/email.dart';
 import 'package:tmail_ui_user/features/mailbox_dashboard/data/model/composer_cache.dart';
 
 abstract class SessionStorageComposerDatasource {
-  void saveComposerCacheOnWeb(Email email);
+  void saveComposerCacheOnWeb(Email email, {required AccountId accountId, required UserName userName});
 
-  ComposerCache getComposerCacheOnWeb();
+  ComposerCache getComposerCacheOnWeb(AccountId accountId, UserName userName);
```

Comment:
Should be return Future< ComposerCache >

Reply:
idem

Reply:
Done